apps/passport-client/src/dispatch.ts

```diff
@@ -551,13 +551,15 @@ async function sync(state: AppState, update: ZuUpdate) {
      * downloading from a pre-v3 version of encrypted storage
      * {@link SyncedEncryptedStorageV3}
      * */
-    const { pcds, subscriptions } = await downloadStorage();
+    const downloaded = await downloadStorage();
 
-    if (subscriptions) {
-      addDefaultSubscriptions(state.identity, subscriptions);
-    }
+    if (downloaded != null) {
+      const { pcds, subscriptions } = downloaded;
+
+      if (subscriptions) {
+        addDefaultSubscriptions(state.identity, subscriptions);
+      }
 
-    if (pcds != null) {
```

Comment:
would `pcds` ever still be `null` if `downloaded` is non-null? and if so, do we want to set `pcds` to `null` versus skipping the download entirely?

Reply:
Good point, i've added a check to make sure that pcds are not null *as well as* the downloaded storage itself